ecpy_hqc_legacy/instruments/drivers/driver_tools.py

```diff
@@ -133,7 +134,9 @@ def wrapper(self, *args, **kwargs):
                     if i == max_iter:
                         raise
                     else:
-                        print(format_exc())
+                        log = logging.getLogger(__name__)
+                        msg = 'Iterating connection %s/%s' % (i, max_iter)
+                        log.exception(msg)
```

Comment:
We know that this can happen for certain instrument on a regular basis, so I am not sure it is worth to spook the users by logging at the exception level, debug should be sufficient. Furthermore, it may be nice to provide some more infos as currently the user does not even know what instrument issued the log entry. We should at least log the type of the driver even if we cannot get in a reliable way the detailed connection infos.

Reply:
ok changed to debug, and added driver info